js/src/forum/components/LinkStatus.tsx

```diff
@@ -37,66 +29,64 @@ export default class LinkStatus extends Component<IAttrs, IState> {
+      </div>
+    );
+  }
 
-        <div className="Provider-Info">
-          {provider.linked() && (
-            <>
-              <dl>
-                <dt>{app.translator.trans('fof-oauth.forum.user.settings.linked-account.link-created-label')}</dt>
-                <dd>{humanTime(provider.firstLogin())}</dd>
+  iconView() {
+    return (
+      <div className="LinkedAccountsList-item-icon">
+        {icon(this.attrs.provider.icon(), { className: `Provider-Icon Provider-Icon--${this.attrs.provider.name()}` })}
+      </div>
+    );
+  }
 
-                <dt>{app.translator.trans('fof-oauth.forum.user.settings.linked-account.last-used-label')}</dt>
-                <dd>{humanTime(provider.lastLogin())}</dd>
+  statusView() {
+    const provider = this.attrs.provider;
 
-                <dt>
-                  {app.translator.trans('fof-oauth.forum.user.settings.linked-account.identifier-label', {
-                    provider: provider.orphaned() ? provider.name() : app.translator.trans(`fof-oauth.lib.providers.${provider.name()}`),
-                  })}
-                </dt>
-                <dd>
-                  <code>{provider.providerIdentifier()}</code>
-                </dd>
-              </dl>
+    return <ProviderInfo provider={provider} />;
+  }
 
-              <Button className={className} icon={provider.icon()} onclick={() => this.deleteProvider(provider)} loading={this.state.loading}>
-                {app.translator.trans('fof-oauth.forum.unlink')}
-              </Button>
-            </>
-          )}
+  actionView() {
+    const provider = this.attrs.provider;
+    const user = this.attrs.user;
 
-          {!provider.linked() && !provider.orphaned() && (
-            <div onclick={() => (app.fof_oauth_linkingProvider = provider.name())}>
-              <LogInButton
-                className={className}
-                icon={provider.icon()}
-                path={`/auth/${provider.name()}?linkTo=${user.id()}`}
-                loading={this.state.loading}
-              >
-                {app.translator.trans(`fof-oauth.forum.log_in.with_${provider.name()}_button`, {
-                  provider: app.translator.trans(`fof-oauth.forum.providers.${provider.name()}`),
-                })}
-              </LogInButton>
-            </div>
-          )}
+    if (provider.linked()) {
+      return (
+        <div className="LinkedAccountsList-item-actions">
+          <Button
+            className={`Button FoFLogInButton LogInButton--${provider.name()} LogInButton${provider.linked() ? '--linked' : '--unlinked'}`}
+            icon={provider.icon()}
+            onclick={() => this.deleteProvider(provider)}
+            loading={this.state.loading}
+          >
+            {app.translator.trans('fof-oauth.forum.unlink')}
+          </Button>
         </div>
-      </li>
-    );
+      );
+    } else if (!provider.orphaned()) {
+      return (
+        <div className="LinkedAccountsList-item-actions">
+          <Button
```

Comment:
If I understand it correctly, it should keep using `LogInButton`? Because it has no `click` handler and the common `Button` has no logic to parse and open the url in `path`. Then the current version cannot link new accounts now.